modules/EvseV2G/connection.cpp

```diff
@@ -644,8 +663,14 @@ static void ssl_key_log_debug_callback(void* ACtx, int ALevel, const char* AFile
     ctx->hexdumpLinesToProcess = 0;
     ctx->inClientRandom = false;
     ctx->inMasterSecret = false;
-    fputc('\n', ctx->file); /* finish key log line */
+
+    const auto message = '\n';
+    fputc(message, ctx->file); /* finish key log line */
     fflush(ctx->file);
+
+    add_character_to_udp_message(message);
```

Comment:
Newline is not necessary here

Reply:
Please delete here `add_character_to_udp_message(message);` A `\n` is not necessary here for the udp payload